changelog/unreleased/kong/bump-atc-router.yml

```
@@ -0,0 +1,5 @@
+message: |
```

```suggestion
message: >
```

More details: https://yaml-multiline.info/

---

I found most of our change logs use "|" instead of ">".